### DISCUSSION TITLE
Welcome to Stage Whisper!

### DISCUSSION THREAD

**Discussion by crazy4pi314** [Stage-Whisper/Stage-Whisper#35] [Announcements]
## 👋 Welcome!
We’re using Discussions as a place to connect with other members of our community. We hope that you:

- Ask questions you’re wondering about.
- Share ideas.
- Welcome and engage with other community members. Remember that this is a community we
 build together 💪.

To get started, check out the post categories on the left, file a [bug](https://github.com/Stage-Whisper/Stage-Whisper/issues/new/choose), or chat with folks on the [Discord](https://discord.gg/Kj8GYSxWGd).

**Comment by ifuchs** [comment]
Is it planned for the app to permit one to go from the transcription to the audio and vice versa? In other words, it would great if there were a way to view the transcription, lick on a word and go to that location to play the audio. Likewise, if I play the audio (after it is transcribed), I'd like to see where I am in the transcription.

**Reply by sawhney17** [reply]
@ifuchs You can't click a specific word but you can click a specific line and play it.

**Reply by ifuchs** [reply]
Close enough:-)

**Reply by petersterne** [reply]
Currently, each line is treated as a separate audio file. So you can click on any line to play just that line. You can also play the whole audio file from the beginning, though (as of now) the lines aren't highlighted as they're played. This is slightly different from Otter's interface, which plays the full audio and highlights each line as it's played, while also letting you click on any line to skip to that point in the audio. 

The upshot is that if you click on a line in Otter, it will play that line and then continue on to the next line and the next line until you stop it. But if you click on a line in Stage Whisper, it will just play that line and then stop.
